datasets/json/json.py

```diff
@@ -12,23 +15,30 @@
 class JsonConfig(nlp.BuilderConfig):
     """BuilderConfig for JSON."""
 
-    read_options: paj.ReadOptions = paj.ReadOptions()
-    parse_options: paj.ParseOptions = paj.ParseOptions()
+    features: nlp.Features = None
+    field: str = None
+    use_threads: bool = True
+    block_size: int = None
+    newlines_in_values: bool = None
```

Comment:
Do you think we could also add the `encoding` as a parameter ?

Reply:
You mean when we default on python reading logic?
I'm tented to always have `utf-8` as default in all the scripts but maybe we could have that as a generic arguments 🤔